src/util/compareThought.ts

```diff
@@ -52,6 +53,17 @@ export const compareStringsWithEmoji = (a: string, b: string) => {
   return aStartsWithEmoji && !bStartsWithEmoji ? -1 : bStartsWithEmoji && !aStartsWithEmoji ? 1 : 0
 }
 
+/** A comparator that sorts meta-attributes above emojis. */
```

Comment:
This comment isn't quite right. The comparator sorts meta attributes above everything, not just emojis. `compareReasonable` is responsible for ordering them.